### DISCUSSION TITLE
nora app under maintenance ?

### DISCUSSION THREAD

**Discussion by iw2lsi** [andrei-tatar/node-red-contrib-smartnora#217] [General]
Hi all...

   is nora under maintenance ? my device is not connected and google home on iOS keep saying that NORA is under maintenance...

        Giampaolo

**Comment by xselvasx** [comment]
i have a similar issue, from node-red everithing is connected but google said that light or blinds are not available

**Comment by andrei-tatar** [comment]
If you are talking about the old NORA (https://node-red-google-home.herokuapp.com/), it has been discontinued and it's not maintained anymore. There are some security vulnerabilities that I don't have the time to patch.

The new one hasn't been down https://smart-nora.eu/ but it does require a subscription for more than 5 devices.

**Reply by xselvasx** [reply]
I'm using the new smart nora, i paid for the annual subscription because i have 12 device, everithing work fine until a few days ago.
Now when i deploy the project in node-red i see in smart-nora.eu all device online but after less than 1 minute everithing became offline, sometimes return online for 10-15 seconds but after return offline; but in node-red seems to be connected. when i try to switch on a light from google voice command it said that the light is not available.
![image](https://user-images.githubusercontent.com/59162073/199681176-cf8366d4-09ff-4e69-b30a-fa3bf0850321.png)
![image](https://user-images.githubusercontent.com/59162073/199681532-8221ea73-34b7-4765-af27-031dbe2ed8f2.png)

**Reply by andrei-tatar** [reply]
@xselvasx what is your user id? I can have a look through the logs. 

Reporting an issue from the menu includes the user id in the description. You can also get it from the node-red logs.

![image](https://user-images.githubusercontent.com/6473183/199682495-c13f8015-9154-426d-8d8f-44e204341466.png)

**Reply by xselvasx** [reply]
issue opened

**Comment by iw2lsi** [comment]
Hi Andrei...

   thanks for your reply...

   you are right... I was still linked with the old NORA... and yes... I
have just one device...

         Giampaolo


Il giorno gio 3 nov 2022 alle ore 09:50 Andrei Tatar <
***@***.***> ha scritto:

**Reply by andrei-tatar** [reply]
In that case, it should be simple and easy to migrate to the new NORA
